resources/views/WelcomePage/WelcomePage.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import {PageHeader, Card, Row, Col, Divider} from 'antd'
+import {Link} from 'react-router-dom'
+
+const WelcomePage = () => (
+  <>
+    <div style={{}}>
+      <PageHeader
+        className='site-page-header'
+        title='Welcome to Cycl.io' />
+    </div>
+    <Divider>Overview</Divider>
+
+    <Row justify='space-around'>
+      <Col span={8}>
+        <Card hoverable title='Inventory Overview' bordered extra={<Link to='/Production/Inventory'>Open</Link>} style={{width: 400}}>
```

Comment:
let's not put the extra here, once the page is resized, the extra is gone, we can put it like the one in the wireframe where it's a footer instead

Reply:
or we can do the card onClick to go to the page as well